### ISSUE TITLE
It looks like the project is abandoned and therefore it's promotions must be stopped

### ISSUE THREAD

**Issue by ghost** [awslabs/aws-shell#266]
`aws-shell` is abandoned. The last update was 2 years ago.
It doesn't support `aws-cli v2`. As a result some people decide to use the older version of `aws-cli`.
But many of `aws-shell` features are implemented in `aws-cli v2` by itself.

I lost several hours before I got understanding that I should use just `aws-cli v2` instead. There are many like me. For example, look at the issue https://github.com/awslabs/aws-shell/issues/238. Two years of never-ending struggle.

I think `aws-shell` should be marked as abandoned/deprecated. It should be removed from pages like https://aws.amazon.com/cli/ where it is promoted as _Developer preview_  - something from the future with active development not from the past with zero activity for the last 2+ years. It's confusing a lot.

**Comment by logikonabstractions** [comment]
Same here. This is ridiculous. The idea is great - because frankly working with aws-cli is not much fun. It feels like a terminal designed in the 90s. One would think the biggest cloud provider could afford to provide decent tools to developers.

**Comment by omenking** [comment]
Okay, so I thought this project was active because its listed right at the top of the AWS Documentation for AWS CLI:

![image](https://user-images.githubusercontent.com/7776/201166431-751c777f-2bca-4123-9143-1b640b61fa10.png)

I was working on free learning content creating a lab and videos for AWS-Shell, but now I realize that the same functionality can be done with: `aws --cli-auto-prompt`

![image](https://user-images.githubusercontent.com/7776/201166665-93e19c7a-abb1-4a53-acfb-2373249964cc.png)

AWS, please remove it from the documentation if this isn't going to be a properly maintained project because it was quite misleading for me and I was able to just mislead a bunch of folks.

**Comment by huyz** [comment]
Frankly, open source developers have a responsibility to make it clear to users when their projects become abandoned.

**Comment by hyandell** [comment]
I agree. I'm going to contact internally within AWS to get attention and presumably to archive/EOL this repository.

**Comment by jeslinmx** [comment]
> I was working on free learning content creating a lab and videos for AWS-Shell, but now I realize that the same functionality can be done with: `aws --cli-auto-prompt`

Thanks for this! Worth noting that you can have this automatically trigger whenever you run a plain `aws` or an otherwise incomplete command by adding this to your `.aws/config`:

```
[default]
...
cli_auto_prompt = on-partial
...
```

Not quite as fast or seamless as having a dedicated shell (which could literally just be `aws --cli-auto-prompt` in a REPL), but oh well.
